codecov/settings_base.py

```diff
@@ -57,6 +57,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "api.shared.middleware.ServiceMiddleware",
```

Comment:
Since this applies to all views (I think) maybe move this into `core.middleware`?